config.json

```diff
@@ -1553,6 +1553,21 @@
           "strings"
         ]
       },
+      {
+        "slug": "react",
+        "name": "React",
+        "uuid": "33531718-1d8a-4abd-bd2a-090303ad0c39",
+        "practices": [
+          "classes",
+          "generic-types"
+        ],
+        "prerequisites": [
+          "classes",
+          "generic-types"
+        ],
```

Comment:
I would probably replace `classes` with `interfaces` as a prerequisite, since that concept is vital for this exercise right now. No need to have both as prerequisites either, since one requires the other.

So I suggest the following:
- Prerequisites: `interfaces`, `generic-types`
- Practices: `classes`, `interfaces`, `generic-types`

Reply:
With the latest commit, I ended up leaving this one for now as the interfaces were removed.